### DISCUSSION TITLE
What does CORTX stand for?

### DISCUSSION THREAD

**Discussion by johnbent** [Seagate/cortx#366] [General] [answered]
We get asked this a lot.  It's not actually an acronym.  Marketing just thought it looks cool and sounds cool.  :smiley:

But I suggest we retroactively try to turn it into an acronym.   I'll go first:
Capacity Optimized Resilient . . . . Technology(?) . . .  eXtraordinaire.

Hmm.  I'll bet someone else can do better than that.  @nikitadanilov ?

**Comment by nikitadanilov** [comment, marked as answer]
Core Object stoRe TeXnology. Or just CORe TeXnology.

**Reply by andriytk** [reply]
Now, let's start the "Core Object" hunt! :)

**Comment by gregnsk** [comment]
Community Of Researchers, Technologists, eXperts

**Comment by andriytk** [comment]
Coherent Object stoRe Technology eXpertise :)

**Comment by iand1111** [comment]
Cloud Object Repository Tuned for eXtreme performance

**Comment by iand1111** [comment]
Cloud Object Repository Tuned for eXascale

**Comment by andriytk** [comment]
Corporate Object stoRe Technology for eXascale

**Comment by andriytk** [comment]
Coherent Object stoRe Technology for eXascale

**Comment by iand1111** [comment]
combining ideas with Andriy's
Cloud Object stoRe Technology for eXascale

**Comment by justinzw** [comment]
Does it have to be storage related? How about Cats Or Rabbits Training Xylophone.  : -)  

or maybe a set of adjectives -> 

Creatively Optimized and Radically Tested Xtreme ( Software Storage Solution)

**Comment by sachinpunadikar** [comment]
sCalable Object stoRe TeXnology

**Comment by mukul-seagate11** [comment]
Core Technology Extension for Object Store

**Comment by vaibhavparatwar** [comment]
Comeon Open the Repository for Technology Xperts!

**Comment by ydb242** [comment]
Comprehensive Object Rich Technology

**Comment by suvratjoshi** [comment]
C-O-R-T-X
**C**atered **O**b-store **R**ationale **T**o e**X**cel
